### PR TITLE
Add PROCESSES document

### PR DESCRIPTION
**Pull request by torstenwalter**
I used this syntax in CODEOWNERS:

```
/chart/<name-of-chart> @maintainer
```

It matches any files in the chart directory at the root of the repository and any of its  subdirectories.
Without the leading `/` it would also match directories found somewhere else. It's unlikely that those names would be used, but it does not harm to do it this way.

Part-of: #38

### CODE REVIEW

**Review by monotek — APPROVED** [review]

File: CODEOWNERS
Comment (scottrigby) [inline CODEOWNERS:1]: This file already exists at /.github/CODEOWNERS. See #12

File: PROCESSES.md
Comment (scottrigby) [inline PROCESSES.md:1]: This processes file looks great 👌

**Review by scottrigby — CHANGES_REQUESTED** [review]

File: PROCESSES.md
Comment (scottrigby) [inline PROCESSES.md:5]: Perhaps it would help to also add sections for responsibilities of chart maintainers and repo admin roles? Per chat and your note about it as well here? https://github.com/prometheus-community/helm-charts/issues/38#issuecomment-687651538

File: PROCESSES.md
Comment (scottrigby) [inline PROCESSES.md:9]: I'm thinking we should link to the existing /.github/CODEOWNERS file from here, so no one misses it again 🙂

**Review by scottrigby — CHANGES_REQUESTED** [review]

File: CODEOWNERS
Comment (torstenwalter) [reply CODEOWNERS:1]: I overlooked that one! Only saw it in #1. Di not notice that it's in a different location. The existing one is better then this. I'll adjust the PR.

File: PROCESSES.md
Comment (torstenwalter) [reply PROCESSES.md]: It's linked now.

File: PROCESSES.md
Comment (torstenwalter) [reply PROCESSES.md:5]: That's a good idea. I would love to split it up into smaller PRs as it's easier to review and get it merged.

**Comment by torstenwalter** [comment]
@scottrigby Thanks for the discussion on slack. I totally overlooked that there was already a CODEOWNERS file in `main` branch. Just in a different folder. I updated this PR with your suggestions.

File: .github/CODEOWNERS
Comment (scottrigby) [inline .github/CODEOWNERS:23]: Just checking, did @acondrat approve @zanhsieh to co-maintain the prometheus-redos-exporter chart?

File: .github/CODEOWNERS
Comment (scottrigby) [inline .github/CODEOWNERS:7]: 💯

File: .github/CODEOWNERS
Comment (scottrigby) [inline .github/CODEOWNERS:9]: Nice catch on the preceding slash. As you said it's unlikely but possible for nested dirs in other charts to have one of these same names. Better to be explicit 👍

File: .github/CODEOWNERS
Comment (acondrat) [reply .github/CODEOWNERS:23]: yes - https://github.com/prometheus-community/helm-charts/pull/46

**Comment by torstenwalter** [comment]
Should we merge this to have CODEOWNER up to date and a first version of processes in the repository?
We can make further enhancements in separate PRs.

**Comment by gkarthiks** [comment]
@torstenwalter  @scottrigby  regarding this PR on CODEOWNERS, I am waiting for approvals from chart owners to add myself as additional maintainers for
|Chart|PR|
|:--:|:--:|
| kube-prometheus-stack|https://github.com/prometheus-community/helm-charts/pull/53 |
|prometheus-consul-exporter|https://github.com/prometheus-community/helm-charts/pull/50|

**Comment by torstenwalter** [comment]
> @torstenwalter @scottrigby regarding this PR on CODEOWNERS, I am waiting for approvals from chart owners to add myself as additional maintainers for
> 
> Chart	PR
> kube-prometheus-stack	#53
> prometheus-consul-exporter	#50

@gkarthiks I saw those. Could you modify the CODEOWNERS directly in those PRs? If chart maintainers approve then we have both in one PR. Depending on what get's merged first we then simply merge main into the still open PRs and resolve conflicts there.

Does this make sense?

**Comment by scottrigby** [comment]
@torstenwalter yep 👍 @acondrat confirmed so we are all good

@gkarthiks soinds good let's make this follow-up PRs once confirmed by those chart owners

**Review by scottrigby — APPROVED** [review]

**Comment by gkarthiks** [comment]
@scottrigby I will do as suggested by @torstenwalter. Will add myself to the CODEOWNERS file as well in the corresponding PRs.

**Pull request merged**
